### PR TITLE
[MBL-16307][Student][Teacher] Merge WebViewExtensions and HtmlContentFormatter

### PR DESCRIPTION
**Pull request by kristofnemere**
refs: MBL-16307
affects: Student, Teacher
release note: none

test plan: check the following screens
- Assignment
- Calendar event
- Discussion
- Page
- Quiz

## Checklist

- [x] Follow-up e2e test ticket created or not needed
- [x] A11y checked
- [x] Approve from product or not needed

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: apps/student/src/main/java/com/instructure/student/fragment/CalendarEventFragment.kt [hunk 1]
```diff
@@ -224,22 +223,17 @@ class CalendarEventFragment : ParentFragment() {
...
+                    calendarEventWebView.setBackgroundColor(ContextCompat.getColor(requireContext(), R.color.backgroundLightest))
+                    calendarEventWebView.loadHtml(html, it.title, baseUrl = scheduleItem?.htmlUrl)
+                }) { url ->
+                    LtiLaunchFragment.routeLtiLaunchFragment(requireContext(), canvasContext, url)
+                }
             }
         }
     }
 
-    private fun loadCalendarHtml(html: String, contentDescription: String?) {
```
Comment (tamaskozmer) [inline apps/student/src/main/java/com/instructure/student/fragment/CalendarEventFragment.kt:237]: I would have left this function, it's cleaner to read than inlined in that loadHtmlWithIframes call.

**Review by tamaskozmer — APPROVED** [review]
Nice work!

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request merged**